### PR TITLE
add khaki skin

### PR DESCRIPTION
**Pull request by kunalvishnoi**
With reference to issue #2 , I have added a khaki skin.

### CODE REVIEW

**Comment by itaditya** [comment]
Attach a  screenshot for preview

**Comment by kunalvishnoi** [comment]
screenshot 103

**Comment by itaditya** [comment]
@kunalvishnoi, one last thing, always remember to add a newline at the end of every file. This is because then only git understands the end of file. Notice that Github is also pointing it .
image
Do read more about it. For now just make this small change and properly format the code(make sure of indentation, spaces after comma etc,)
:smile:

File: assets/skins/khakicircle.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  border-color: rgba(255,255,255,0.4);
+  box-shadow: 2px 2px 20px 0 rgba(0,0,0,0.6);
+}
+.khaki .text {
+  color: #fff68f !important;
+  font-weight: bold;
+}
+.khaki .loader-spinner {
+  border-color: #fff68f;
+}
```
Comment (itaditya) [inline assets/skins/khakicircle.css]: Add a newline

Comment (kunalvishnoi) [reply assets/skins/khakicircle.css]: new line add in the last of file

**Review by itaditya — APPROVED** [review]
Good work

**Pull request merged**